### DISCUSSION TITLE
taro在请求资源跨域问题

### DISCUSSION THREAD

**Discussion by Lier1210** [NervJS/taro#13902] [Q & A]
我现在是taro的使用的一个开发者，我是真的想知道，在使用时，在config/dev内配置了proxy代理，但是请求数据时并没有效果；
还有一点，我想知道，taro.reques()，这个方法，在设置mode：cors，貌似也没有效果，我感觉到身心疲惫，不想在使用了

**Comment by ZakaryCode** [comment]
taro 并没有在 config 下提供 proxy 能力，如果需要配置 webpack 代理，可以通过 webpackChain 完成，详细参考对应[文档](https://nervjs.github.io/taro-docs/docs/config-detail#h5webpackchain)；
询问方法请注明使用平台，便于快速定位，考虑到所述问题应当是 h5 平台，request API mode 配置会直接绑定到请求头上，实际效果与浏览器环境和服务端接口关联，这需要参考 [MDN 官方文档](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)并仔细查看调用接口返回头与请求头实际参数。
